### DISCUSSION TITLE
跳脫手套的 VR 觸覺方案，可穿戴的電子「皮膚」

### DISCUSSION THREAD

**Discussion by RogelioKG** [1101-GS3073-C/Python-Community-GS3073-C#586] [General]
Content:
https://mag.addmaker.tw/2022/12/26/vr-wetac/

Reflections / Thoughts:
就像是電影《 一級玩家》中，頭戴 VR 顯示器、身穿體感觸控衣的主角，進入元宇宙世界的標準配備，應該要能產生跟現實世界一樣的各種感官體驗。WeTac 就是一個很不錯的體感觸控技術，它最大的優點在於它相當輕便，而不需要像其他體感模擬的手套相當笨重甚至是有線的。

**Comment by Raywei0214** [comment]
感覺挺酷的，但有nervgear應該就用不到了

**Comment by manipulateed** [comment]
觸覺身為人很重要的知覺系統，透過這種穿戴式裝置，多少還是有點落差，未來固然會發展出更精細更符合人原始感覺的工具，期待未來的發展。

**Comment by awkward-willy** [comment]
這樣的穿戴式裝置非常創新，很想實際體驗看看，畢竟能真實地觸摸到虛擬的東西能大幅增加使用者的體驗並營造沉浸感。期望看到這項科技在未來被實踐在各大領域，同時也疑惑這樣的產品會不會有悶熱、不舒服等問題有待改善。

**Comment by larry40305** [comment]
我相信nevgear很快就會真的發明出來了

**Comment by magicmicheal** [comment]
感覺這在醫學界是個很大的突破，有些人受到燒傷等因素，可能導致喪失了觸覺，這項技術可能能十分有效的幫助他們

**Comment by 0212Jerry** [comment]
這是一個很酷的發明，現今我們的元宇宙、虛擬世界已經有一個雛形了，可以滿足人類對於視覺上的各種娛樂，但在觸覺上我們難以打造，無法在虛擬世界裡體驗各種觸覺，這個發明的出現無疑是讓虛擬世界更完整。

**Comment by Vincent20030314** [comment]
讓玩家更有遊戲體驗，更能模擬遊戲的真實性，真是不錯的發明。

**Comment by Valentinaa0214** [comment]
超酷的！若是真的能夠實際仿真真實感受、帶給使用者真實感受的觸覺回饋，虛擬世界與真實世界的界線感會大幅降低，這將會是人類前進元宇宙的一大步。
